docker-compose.yml

```diff
@@ -22,6 +22,11 @@ services:
       - db
     stdin_open: true
     tty: true
-
+  node:
+    image: node:9-alpine
+    working_dir: /node
+    volumes:
+      - .:/node
+    command: /bin/sh bin/docker-node.bash
```

Comment:
If you're going the `sh` route (I know, it's the default shell in Alpine), you might as well rename `docker-node.bash` to `docker-node.sh` and add the appropriate shebang to the top of the shell script:  `#!/bin/sh`.  Then you can replace `/bin/sh` with `bin/docker-node.bash`.

Reply:
Ah that makes sense! On it